.gitlab/ci/global.yml

```diff
@@ -13,67 +13,69 @@
     policy: pull
 
 
-.default-before_script:
-  before_script:
-    - source .gitlab/helper_functions.sh
-    - echo "=== Running before script ==="
-    - git checkout $CI_COMMIT_BRANCH
-    - git config diff.renameLimit 6000
-    - mkdir -p -m 777 $ARTIFACTS_FOLDER/
-    - |
-      if [[ -f "$BASH_ENV" ]]; then
-        source "$BASH_ENV"
-      fi
-    - source .circleci/content_release_vars.sh
-    - export PATH="${PWD}/node_modules/.bin:${PATH}"
+.prepare_env: &prepare_env
+  - source .gitlab/helper_functions.sh
+  - echo "=== Running before script ==="
+  - git checkout $CI_COMMIT_BRANCH
+  - git config diff.renameLimit 6000
+  - mkdir -p -m 777 $ARTIFACTS_FOLDER/
+  - |
+    if [[ -f "$BASH_ENV" ]]; then
+      source "$BASH_ENV"
```

Comment:
Do we even have a `BASH_ENV` file in our GitLab pipelines? I thought this was a CircleCI [feature](https://circleci.com/docs/2.0/env-vars/#using-parameters-and-bash-environment).